app/src/main/res/values/preference_keys.xml

```diff
@@ -217,6 +217,11 @@
 
     <string name="pref_key_open_tabs_count" translatable="false">pref_key_open_tabs_count</string>
 
+    <string name="pref_key_installed_addons_count" translatable="false">pref_key_installed_addons_count</string>
+    <string name="pref_key_installed_addons_list" translatable="false">pref_key_installed_addons_list</string>
+    <string name="pref_key_enabled_addons_count" translatable="false">pref_key_enabled_addons_count</string>
+    <string name="pref_key_enabled_addons_list" translatable="false">pref_key_enabled_addons_list</string>
```

Comment:
It would be nice to be able to group all these telemetry count/list pref key under a particular header. I find myself also adding comments to pref keys if I find them to be helpful.

Reply:
Sure, I'll add a comment above this group